providers/latest-provider/actions.js

```diff
@@ -33,9 +33,11 @@ export const getLatest = createThunkAction(
                   // Can this service return "null" or similar and then we can handle that case here
                   const days = statsLatestDecoder(bands);
                   // convert to date
-                  date = moment('2014-12-31')
-                    .add(days, 'days')
-                    .format('YYYY-MM-DD');
+                  date = days
+                    ? moment('2014-12-31')
```

Comment:
could we add this date as a constant @01painadam ?

Reply:
yes, it can